express-delete/express-delete/index.js

```diff
@@ -0,0 +1,39 @@
+const express = require('express');
+const app = express();
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+app.listen(3000, () => {
+});
```

Comment:
Although this will work fine here, it is best to have it as the last thing in the file. You should also have a console log in the callback so that you will know your server started correctly and the port number the server is running on.